content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -497,6 +502,13 @@ If you want to use project quotas, you should:
   [kubelet configuration](/docs/reference/config-api/kubelet-config.v1beta1/)
   or the `--feature-gates` command line flag.
 
+* Enable the `UserNamespacesSupport=true`
```

Comment:
this should be `Ensure the `UserNamespacesSupport` feature gate is enabled, and that your kernel, CRI implementation and OCI runtime support user namespaces`, as it's enabled by default now